tests/integration/telemetry/stats/prometheus/wasm/bad_wasm_filter_test.go

```diff
@@ -0,0 +1,115 @@
+// +build integ
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package wasm
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	"golang.org/x/sync/errgroup"
+
+	resource "istio.io/istio/pilot/pkg/xds/v3"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/util/retry"
+	util "istio.io/istio/tests/integration/telemetry"
+	common "istio.io/istio/tests/integration/telemetry/stats/prometheus"
+)
+
+// TestBadWasmRemoteLoad tests that bad Wasm remote load configuration won't affect service.
```

Comment:
nit: it would be better to add the overall workflow in the comments as well

Reply:
Done.